lte/gateway/c/core/BUILD.bazel

```diff
@@ -1018,17 +1018,17 @@ cc_library(
         "oai/tasks/s11/s11_mme_bearer_manager.h",
         "oai/tasks/s11/s11_mme_session_manager.h",
         "oai/tasks/s1ap/3gpp_requirements_36.413.h",
```

Comment:
Bazel build job is failing because the file `lte/gateway/c/core:oai/tasks/s1ap/3gpp_requirements_36.413.h` was removed but is still listed in the build file - could we remove it here as well?